crates/bevy_asset/src/path.rs

```diff
@@ -63,6 +63,51 @@ impl<'a> AssetPath<'a> {
                 .map(|value| Cow::Owned(value.to_string())),
         }
     }
+
+    /// Resolves a relative asset path. The result will be an `AssetPath` which is resolved relative
+    /// to this path. There are three cases:
```

Comment:
Nit: I'd split the "There are three cases:" out of this first paragraph

Reply:
OK

Reply:
Done